definitions/cross_product.toml

```diff
@@ -0,0 +1,53 @@
+[metrics]
```

Comment:
Naming is hard; happy to rename or consider alternatives for this filename if there are any strong opinions out there.

Reply:
`multi_product` was my thought as well, and I think that'd be my vote, but don't feel strongly about it.

Reply:
I like `multi_product` and hadn't considered it. I'll make that change.